changelog/1274.breaking.rst

```diff
@@ -0,0 +1,6 @@
+Moved the charged particle radiography analysis codes into a new module
+`~plasmapy.diagnostics.charged_particle_radiography` containing synthetic
+radiograpy tools in
+`~plasmapy.diagnostics.charged_particle_radiography.synthetic_radiography`
+and detector stack calculation tools in
+`~plasmapy.diagnostics.charged_particle_radiography.detector_stacks`
```

Comment:
```suggestion
Moved the charged particle radiography analysis codes into a new module `~plasmapy.diagnostics.charged_particle_radiography` containing synthetic radiography tools in `~plasmapy.diagnostics.charged_particle_radiography.synthetic_radiography` and detector stack calculation tools in `~plasmapy.diagnostics.charged_particle_radiography.detector_stacks`.
```
One *possible* cause of the doc failure is that it is interpreting this as a list or something.  Maybe.  In the past I think I was able to fix something like this by putting the changelog entry all on one line.  Not sure why.  But, could be worth trying out.